src/zope/configuration/xmlconfig.py

```diff
@@ -201,11 +205,9 @@ def startElementNS(self, name, qname, attrs):
 
         try:
             self.context.begin(name, data, info)
-        except (KeyboardInterrupt, SystemExit): # pragma: no cover
+        except self.testing:
```

Comment:
This seems too clever by half.

Reply:
To clarify, I don't like the naming, not the logic.

Reply:
I agree, I tied myself in knots keeping BWC with the name, all to avoid a single `if` on no good basis. I'll just go back to the simple thing.